packages/xo-web/src/xo-app/logs/backup-ng/log-alert-body.js

```diff
@@ -340,7 +354,7 @@ export default decorate([
       },
     },
     computed: {
-      log: (_, { log }) => {
+      log: (_, { log, vms = {} }) => {
```

Comment:
Why the default value for `vms`?

Reply:
No need `createGetObjectsOfType` return empty object on fetching.